web-marketplace/src/pages/Dashboard/MyEcocredits/MyEcocredits.tsx

```diff
@@ -62,12 +60,13 @@ import useOpenTakeModal from './hooks/useOpenTakeModal';
 import { useUpdateCardItemsTakeBasket } from './hooks/useUpdateCardItemsTakeBasket';
```

Comment:
missing translation for `BASKET_TAKE_TITLE` and `CREDIT_SEND_TITLE`
will the various forms (eg send credits) be handled separately?

Reply:
Yes, it will be handled separately.